wiki/PlaceholderExpansion.md

```diff
@@ -58,19 +58,27 @@ public class SomeExpansion extends PlaceholderExpansion {
 Let's quickly break down the different methods you had to import.
 
 - #### getAuthor
-  Through this method can you set the name of who created the expansion.
+  Through this method you can set the name of who created the expansion.
 - #### getIdentifier
   The name that should be used to identify the placeholders for this expansion.  
   The Identifier is the first text after the `%` and before the first `_` (`%identifier_values%`) and can therefore not contain any `_` in it.
   
   If you want to use `_` in your Expansion's name can you override the optional `getName()` method.
 - #### getVersion
-  This is a String, which means it doesn't has to be a number in itself. The String is used to determine if a new update is available or not, when the expansion is shared on the eCloud.  
-  For Expansions that are part of a plugin does this not really matter.
+  This is a String, which means it doesn't have to be a number in itself. The String is used to determine if a new update is available or not, when the expansion is shared on the eCloud.  
+  For Expansions that are part of another plugin does this does not really matter.
 
 Those are all the neccessary parts for your PlaceholderExpansion.  
 Any other methods that are part of the [`PlaceholderExpansion`][placeholderexpansion] class are optional and will usually not be used, or default to a specific value. Please read the Javadoc comments of those methods for more information.
 
+There are two specific methods which are options, but you should use at least one of them:
+
+- #### onRequest(OfflinePlayer, String)
+  If not set this method will default to calling [`onPlaceholderRequest(Player, String)`](#onplaceholderrequestplayer-string).  
+  This method is recommended to use as it allows the usage of `null` and can therefore be used in placeholders that don't need a valid player to be used.
+- #### onPlaceholderRequest(Player, String)
+  If not set this method will return `null` which PlaceholderAPI sees as invalid placeholder.
+
 ----
 ## Without a Plugin
 A PlaceholderExpansion may not need a plugin to rely on, if the placeholders it provides can return values from just the server itself or some other source (i.e. Java itself).
```

Comment:
```suggestion
An expansion does not always need a plugin to rely on. If the placeholders it provides can return values from just the server itself or some other source (i.e. Java itself), then it can work independently.
 
```